### ISSUE TITLE
iOS 13 Beta + Stripe (17.0.0) --> if device is not on dark mode, the nav controller that says "Add Card" is not displaying correctly

### ISSUE THREAD

**Issue by kselvin** [stripe/stripe-ios#1369]
Stripe 17.0 
device model -- 7 plus, 7, and X 
iOS : 13.1 Beta 2 

If dark mode is on, the add card vc looks fine, BUT if dark mode is OFF see image 

![image0](https://user-images.githubusercontent.com/14080338/64638624-22177180-d3d4-11e9-9d47-19643d72dbd4.png)

Code to present view controller:  
```
self.paymentContext!.hostViewController = self.navigationController
self.paymentContext!.presentPaymentOptionsViewController()
```

note:  If i download from app store on iOS 12 everything looks ok, this result is only on iOS 13 devices

any help on this would be greatly appreciated!!! (my app is live in the app store and im afraid people will start updating soon)

thank you!

**Comment by davidme-stripe** [comment]
This is a dupe of #1350. It's fixed in the master branch of the Stripe SDK, and we're planning to release an update to Cocoapods later today.

One thing: I think this only happens when you build with the iOS 13 SDK — I don't think UIKit will use the new navigation bar style if someone downloads your app from the App Store. I could be wrong on this, please let me know!

**Comment by kselvin** [comment]
Great thank you! 

I am pretty sure it will show in App Store because i created a build on Fabric using iOS 12 SDK and then installed it on iOS 13 devices to find the bug

**Comment by kselvin** [comment]
Followed the issue you linked to and was able to solve the issue using  
self.paymentContext!.theme.translucentNavigationBar = true  

thank you!!

**Comment by davidme-stripe** [comment]
Great! The update has shipped, so you can also work around this by updating to Stripe 17.0.1.

**Comment by aprashar3** [comment]
In your app did finish launching with options method add the following lines of code. Change the colour property as required. It worked for me hope it work for others also.

Obj-c

if (@available(iOS 13.0, *)) {
        UINavigationBarAppearance *appearance = [[UINavigationBarAppearance alloc] init];
        [appearance configureWithOpaqueBackground];
        appearance.backgroundColor = UIColor.whiteColor;
        [appearance setTitleTextAttributes:@{NSForegroundColorAttributeName:[UIColor blackColor]}];
        [appearance setLargeTitleTextAttributes:@{NSForegroundColorAttributeName:[UIColor blackColor]}];
        UINavigationBar.appearance.scrollEdgeAppearance = appearance;
        UINavigationBar.appearance.standardAppearance = appearance;
    }

Swift

let coloredAppearance = UINavigationBarAppearance()
coloredAppearance.configureWithOpaqueBackground()
coloredAppearance.backgroundColor = .white
coloredAppearance.titleTextAttributes = [.foregroundColor: UIColor.black]
coloredAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.black]
       
UINavigationBar.appearance().standardAppearance = coloredAppearance
UINavigationBar.appearance().scrollEdgeAppearance = coloredAppearance

Found a great link regarding the same : https://sarunw.com/posts/uinavigationbar-changes-in-ios13/
